src/clipboard.ts

```diff
@@ -72,17 +73,34 @@ export const copyToClipboard = async (
         }, {} as BinaryFiles)
       : undefined,
   };
-  const json = JSON.stringify(contents);
-  CLIPBOARD = json;
+
+  const clipboardDataAsText = transformClipboardElementsToText(
+    contents.elements,
+  );
+  CLIPBOARD = clipboardDataAsText;
+
   try {
     PREFER_APP_CLIPBOARD = false;
-    await copyTextToSystemClipboard(json);
+    await copyTextToSystemClipboard(clipboardDataAsText);
   } catch (error: any) {
     PREFER_APP_CLIPBOARD = true;
     console.error(error);
   }
 };
 
+export const transformClipboardElementsToText = (
+  elements: readonly NonDeletedExcalidrawElement[],
+): string => {
+  const onlyTextElements = elements.every((element) => element.type === "text");
+  if (onlyTextElements) {
+    return elements
+      .map((element) => normalizeText((element as ExcalidrawTextElement).text))
```

Comment:
this will need to be written. So let's actually do the same with do in `copyText` action:

```
 const text = elements
      .reduce((acc: string[], element) => {
        if (isTextElement(element)) {
          acc.push(element.text);
        }
        return acc;
      }, [])
      .join("\n\n").trim();
```

And then check if `text` is empty.

Also no need to normalize the `element.text` as it's already noramlized.

Reply:
Note that all this should be done in `parseClipboard`, not `copyToClipboard`, which deals with a different feature.